agris.gemspec

```diff
@@ -24,10 +23,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'savon', '~> 2.11'
 
   spec.add_development_dependency 'bundler', '~> 1.14'
+  spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'rake', '~> 10.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'rubocop', '0.49.0'
+  spec.add_development_dependency 'rubocop'
```

Comment:
Version? Is there a reason you did specify the version in hound?

Reply:
I just removed it thinking we should just enforce it with the Gemfile.lock.

But just realized the Gemfile.lock is ignored from the repo. So I'll add it back here.